pkg/database/upsertCluster.go

```diff
@@ -3,12 +3,31 @@ package database
 import (
 	"context"
 	"encoding/json"
+	"fmt"
 	"reflect"
 
 	"github.com/stolostron/search-indexer/pkg/model"
 	"k8s.io/klog/v2"
 )
 
+func (dao *DAO) DeleteCluster(clusterName string) {
+	clusterUID := string("cluster__" + clusterName)
+	// Delete resources for cluster from resources table from DB
+	_, err := dao.pool.Exec(context.Background(), "DELETE FROM search.resources WHERE cluster=$1", clusterName)
```

Comment:
Use context from parent.

Reply:
Should I pass the context from stopAndStartInformer to processClusterUpsert  and processClusterDelete?

Reply:
Added

Reply:
yes, thanks!